app/helpers/application_helper.rb

```diff
@@ -577,4 +577,33 @@ def radio_button_list(form, field_name, option_array)
       end
     end
   end
+
+  # Identifier for creation, formatted external-work-12, series-12, work-12.
+  def creation_id_for_css_classes(creation)
+    return if creation.nil?
+    return unless %w[ExternalWork Series Work].include?(creation.class.name)
+
+    "#{creation.class.name.underscore.dasherize}-#{creation.id}"
+  end
+
+  # Space-separated list of creator ids, formatted user-123 user-126.
+  # External works are not created by users, so we can skip this.
+  # TODO: We will want to list users on series blurbs, but when a series becomes
+  # or stops being anonymous, it is not touched, so the cache in
+  # css_classes_for_creation_blurb will not expire.
+  def creator_ids_for_css_classes(creation)
```

Comment:
Yeah, that sounds good, and it actually brings me to a question I had: since they're only used in `css_classes_for_creation_blurb`, and I really can't see a use for them directly, do we think the `nil?` checks are necessary?

Reply:
Yeah we can live without them.

Reply:
Also I just learned that ["private" doesn't apply to helpers](https://stackoverflow.com/questions/12835367/why-can-private-helper-methods-still-be-accessed-in-views)... ah well.

Reply:
I thought it was a bit strange my tests were still working 😆